Specs/DateTime/Spanish/DatePeriodExtractor.json

```diff
@@ -4434,67 +4433,67 @@
     ]
   },
   {
-    "Input": "Ya terminé todos los trabajos hace más de 2 semanas",
-    "NotSupported": "dotnet, java",
+    "Input": "durante más de 2 semanas antes de hoy",
+    "NotSupported": "java",
     "NotSupportedByDesign": "javascript, python",
     "Results": [
       {
-        "Text": "hace más de 2 semanas",
+        "Text": "durante más de 2 semanas antes de hoy",
         "Type": "daterange",
-        "Start": 30,
-        "Length": 21
+        "Start": 0,
+        "Length": 37
       }
     ]
   },
   {
-    "Input": "Volveré dentro de 2 semanas",
-    "NotSupported": "dotnet, java",
+    "Input": "dentro de 2 semanas a partir de hoy",
```

Comment:
> > **4 cases won't fix by design**
> > 1.Ya no estaré 3/15
> > 2.en los 70
> > 3.los noventa
> > 4.En mil novecientos veintiocho
> 
> The last one should be fixed. The same potentially ambiguous scenario applies in other languages.
> 
> Why not fix `#1`? And should cases `#2` and `#3` be changed to "los años 70" and similar?

#1 type is date not dateperiod
#2 #3 will modify case to "los años" 
#4 will support in next version

Reply:
because this sentences is translated by English DatePeriod Case "I will come back within 2 weeks from today" so I add "a partir de hoy" follow Intern's suggestion who major in Spanish,

Reply:
because this sentences is translated from English DatePeriod Case "I will come back within 2 weeks from today" so I add "a partir de hoy" follow Intern's suggestion who major in Spanish,also 
 "Volveré dentro de 2 semanas" is make sense, will support it